WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -10,7 +12,13 @@ public static class StartupHelper
 	{
 		private const string KeyPath = "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run";
 
-		public static void ModifyStartupSetting(bool runOnSystemStartup)
+		// Arguments to add Wasabi to macOS startup settings.
+		private static readonly string AddCmd = $"osascript -e \' tell application \"System Events\" to make new login item at end with properties {{name:\"WasabiWallet\", path:\"/Applications/WasabiWallet.app\",hidden:false}} \'";
```

Comment:
`/Applications/WasabiWallet.app` needs to be replaced with some dynamic piece of code.

Reply:
Done